.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       uses: actions/checkout@v2
       with:
         repository: UNC-Libraries/Carolina-Digital-Repository.git
-        ref: 359e459ad6c3a02b7aa68d7bfab5a90ad8cfb38f
+        ref: f580221852b673e1af442859d229393e80518b13
```

Comment:
Will this need to be updated every time a boxc pr is merged?

Reply:
It could be set to `main`, but if it needs to reference something that's not merged yet we'd need to specify a commit. It would just need to be updated when it needs a more recent version of boxc, which would unfortunately be for most PRs. I'm not sure how else to handle it.

Reply:
Do you think this would work, https://github.com/actions/checkout#checkout-head?